lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -194,6 +214,54 @@ private void clearComposition() {
     // Do nothing.
   }
 
+  /**
+   * Add a color filter to specific content on a specific layer.
+   * @param layerName non null, name of the layer where the supplied content name lives
+   * @param contentName non null, name of the specific content that the color filter is to be
+   *                    applied
+   * @param colorFilter the color filter
```

Comment:
Same javadocs as interface.
Alternatively, you can only have the javdoc here and use `@see LottieDrawable#add*` as the javadoc in the DrawingContent interface